baybe/_optional/info.py

```diff
@@ -44,7 +43,7 @@ def exclude_sys_path(path: str, /):  # noqa: DOC402, DOC404
 TYPOS_INSTALLED = True
 
 # Package combinations
```

Comment:
The heading no longer fits. Perhaps replace it with `Derived flags`

Reply:
adapted